test/e2e/cws-tests/tests/test_e2e_cspm_docker.py

```diff
@@ -1,150 +0,0 @@
-import json
-import os
-import socket
-import time
-import unittest
-import uuid
-import warnings
-
-from lib.config import gen_datadog_agent_config
-from lib.cspm.api import App
-from lib.docker import DockerHelper
-from lib.stepper import Step
-from test_e2e_cspm import expect_findings
-
-
-class TestE2EDocker(unittest.TestCase):
-    def setUp(self):
-        warnings.simplefilter("ignore", category=ResourceWarning)
-        warnings.simplefilter("ignore", category=UserWarning)
-
-        self.docker_helper = DockerHelper()
-        self.app = App()
-
-    def tearDown(self):
-        self.docker_helper.close()
-
-    def test_privileged_container(self):
-        print("")
-
-        test_id = str(uuid.uuid4())[:4]
-        with Step(msg="create privileged container", emoji=":construction:"):
-            pc = self.docker_helper.client.containers.run(
-                "ubuntu:latest",
-                command="sleep 7200",
-                detach=True,
-                remove=True,
-                privileged=True,
-            )
-            self.container_id = pc.id
-
-        with Step(msg="check agent start", emoji=":man_running:"):
-            image = os.getenv("DD_AGENT_IMAGE")
-            hostname = f"host_{test_id}"
-            self.datadog_agent_config = gen_datadog_agent_config(
-                hostname=hostname, log_level="DEBUG", tags=["tag1", "tag2"]
-            )
-
-            self.container = self.docker_helper.start_cspm_agent(
-                image,
-                datadog_agent_config=self.datadog_agent_config,
-            )
-            self.assertIsNotNone(self.container, msg="unable to start container")
-
-            self.docker_helper.wait_agent_container()
-
-        with Step(msg="check agent events", emoji=":check_mark_button:"):
-            self.container.exec_run("security-agent compliance check --dump-reports /tmp/reports.json --report")
-            _, output = self.container.exec_run("cat /tmp/reports.json")
-            print(output)
-            findings = json.loads(output)
-
-            expected_findings = {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should you port also the docker test?